library/core/src/convert/mod.rs

```diff
@@ -440,7 +440,7 @@ pub trait TryInto<T>: Sized {
 ///
 ///     fn try_from(value: i32) -> Result<Self, Self::Error> {
 ///         if value <= 0 {
-///             Err("GreaterThanZero only accepts value superior than zero!")
+///             Err("GreaterThanZero only accepts values greater than zero!")
```

Comment:
```suggestion
///             Err("GreaterThanZero only accept values greater than zero!")
```
Not sure which is grammatically correct?

Reply:
I think the way it's written is correct.